### PR TITLE
add compat data for SourceBuffer

### PR DESCRIPTION
**Pull request by gsouquet**

### CODE REVIEW

File: api/SourceBuffer.json [hunk 1]
```diff
@@ -0,0 +1,1306 @@
...
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
```
Comment (jpmedley) [inline api/SourceBuffer.json]: Add Chrome and Opera, all with the value `false`.

**Comment by gsouquet** [comment]
@jpmedley fixed, thanks for you feedback. 
Again, may I ask where you find all those compat data? Maybe in the future I could cross check what is on MDN with whatever the real compat data is. 

Thanks!

**Comment by jpmedley** [comment]
It's a bit complicated. It involves digging through commit histories and emails with feature owners. I've been meaning to write an article about it for more than a year, but haven't found the time. We're working on automation, though. Thanks for your offer.

**Comment by gsouquet** [comment]
Right, I try to look at the changelogs but it's not always very clear. 
I can't really easily find my way in the W3C mailing lists... So yeah, i'd be quite keen to read that article you wanna write :)

**Pull request merged**